src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -355,10 +376,10 @@ public OffenderDetail getOffenderDetailByNomsNumber(final @PathVariable("nomsNum
         })
     @GetMapping(path = "/offenders/crn/{crn}/convictions/{convictionId}")
     public Conviction getConvictionForOffenderByCrnAndConvictionId(
-        @ApiParam(name = "crn", value = "CRN for the offender", example = "A123456", required = true)
-        @NotNull @PathVariable(value = "crn") final String crn,
-        @ApiParam(name = "convictionId", value = "ID for the conviction / event", example = "2500295345", required = true)
-        @NotNull @PathVariable(value = "convictionId") final Long convictionId) {
+            @ApiParam(value = "CRN for the offender", example = "A123456", required = true)
+            @NotNull @PathVariable(value = "crn") final String crn,
+            @ApiParam(value = "ID for the conviction / event", example = "2500295345", required = true)
```

Comment:
don't need to specify the `name` and adds scope for error

Reply:
There are ApiParam references in other controllers - are we getting rid of all ?